app/poros/movie.rb

```diff
@@ -1,9 +1,22 @@
 class Movie
-  attr_reader :id, :title, :vote_avg, :overview
+  attr_reader :id, :title
   def initialize(movie_data)
     @id = movie_data[:id]
     @title = movie_data[:title]
-    @vote_avg = movie_data[:vote_average]
-    @overview = movie_data[:overview]
+    @raw_hash ||= MovieDBService.new.find_by(@id)
```

Comment:
@rrabinovitch trying to think through this Movie model and how it works... when we create movie objects right now, they're coming from two different endpoints (and possibly more later on), so im wondering what is redundant and what isn't. In this case, the original `movie_data` hash from a particular endpoing may not have all the info we need, so I figured we could use the `MovieDBService` to pull the rest of the info? 

At that point, the decision is whether to also put it in `initialize`, or make separate methods. I'm not sure what the best option is in order to reduce API calls.